go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/controller-runtime
 
-go 1.22.0
+go 1.22
```

Comment:
To be honest. Not sure what we should do here. We won't be able to get CI green if go mod tidy changes the go mod file. I also don't want to check in a go.mod file that is updated as soon as a go command is run

Reply:
Responded on the issue: https://github.com/kubernetes-sigs/controller-runtime/issues/2720#issuecomment-2016461180 (just saw the comments there now)